src/sdk-reference/csharp/1/collection/get-mapping/snippets/get-mapping.cs

```diff
@@ -0,0 +1,8 @@
+try {
+  string mapping = kuzzle.collection.getMapping("nyc-open-data", "yellow-taxi");
+
+  Console.WriteLine(mapping);
+  // {"properties":{"license":{"type":"keyword"},"driver":{"properties":{"name":{"type":"keyword"},"curriculum":{"type":"text"}}}}}
+} catch (KuzzleException e) {
+  Console.Error.WriteLine(e.getMessage());
```

Comment:
```suggestion
  Console.Error.WriteLine(e.ToString());
```

Reply:
I won't notify about other occurence of that error, but they still need to be fixed.